apis/rds/v1alpha1/custom_types.go

```diff
@@ -122,6 +122,22 @@ type CustomDBClusterParameters struct {
 	// set DBClusterParameterGroupName.
 	// +optional
 	DBClusterParameterGroupNameSelector *xpv1.Selector `json:"dbClusterParameterGroupNameSelector,omitempty"`
+
+	// A value that indicates whether the modifications in this request and any
+	// pending modifications are asynchronously applied as soon as possible, regardless
+	// of the PreferredMaintenanceWindow setting for the DB cluster. If this parameter
+	// is disabled, changes to the DB cluster are applied during the next maintenance
+	// window.
+	//
+	// The ApplyImmediately parameter only affects the EnableIAMDatabaseAuthentication,
+	// MasterUserPassword, and NewDBClusterIdentifier values. If the ApplyImmediately
+	// parameter is disabled, then changes to the EnableIAMDatabaseAuthentication,
+	// MasterUserPassword, and NewDBClusterIdentifier values are applied during
+	// the next maintenance window. All other changes are applied immediately, regardless
+	// of the value of the ApplyImmediately parameter.
+	//
+	// By default, this parameter is disabled.
+	ApplyImmediately *bool `json:"applyImmediately,omitempty"`
```

Comment:
I think having this as `true` in examples would be less surprising to new folks.